fastlane/Fastfile

```diff
@@ -180,14 +180,16 @@ end
 desc 'Upload file to s3'
 lane :upload_file_to_s3 do |options|
   os_type = options[:os_type].downcase
-  extension = os_type == "android" ? "*.apk" : "*.ipa"
+  extensions = os_type == "android" ? ["*.apk", "*.aab"] : ["*.ipa"]
   build_folder_path = Dir[File.expand_path('..')].first
   files = []
   
   unless options[:file].nil? || options[:file].empty?
     files.push("#{build_folder_path}/#{options[:file]}")
   else
-    files = Dir.glob("#{build_folder_path}/#{extension}").select { |f| File.file? f}
+    extensions.each do |extension|
+      files.push(Dir.glob("#{build_folder_path}/#{extension}").select { |f| File.file? f}.first)
```

Comment:
this broke the CI as is only uploading one apk instead of every apk in the output. Will submit a PR to fix

Reply:
Can you also tag me in that PR? @enahum